javascript/async-apis/APIs.md

```diff
@@ -2,6 +2,14 @@
 
 One of the most powerful things a web developer can do is fetching data from a server and displaying it creatively on their site. In many cases, the server solely exists for that specific site. The server could contain blog posts, user data, high scores for a game or anything else. In other cases, the server is an open service that serves data to anyone that wants to use it (i.e. weather data or stock prices). In either case, the methods of accessing and then using that data are essentially the same.
 
+### Learning Outcomes
+By the end of this lesson, you should be able to:
+
+ - Explain what an API is
+ - Explain how access to an API broadly works
```

Comment:
Broadly seems in the wrong place. Did you mean to put it after Explain?

Reply:
I actually started with it where you've suggested it and then changed it 😁 I'll revert.